README.md

```diff
@@ -52,6 +52,43 @@ To verify the installation, find the pod name of the deployed `wavefront-collect
 ```
 kubectl logs -f COLLECTOR_POD_NAME -n wavefront-collector
 ```
+## Installation on OpenShift Origin 3.9
```

Comment:
@srinivas-kandula Can you please move the Openshift specific stuff to `docs/openshift.md` and just link from the main readme?

Reply:
Done, moved steps to new file.